### DISCUSSION TITLE
WARNING! Windows is not officially supported yet. Things may or may not work as expected.

### DISCUSSION THREAD

**Discussion by JAYAMURUGANJ** [serverpod/serverpod#376] [Q&A] [answered]
serverpod create samplepod
after running the above command am getting the follow warnning
WARNING! Windows is not officially supported yet. Things may or may not work as expected.

dart bin/main.dart
SERVERPOD version: 0.9.19 mode: development time: 2022-11-14 21:03:26.735013Z
2022-11-14 21:03:27.150352Z Failed to connect to database.
PostgreSQLSeverity.error 42P01: relation "serverpod_runtime_settings" does not exist

**Comment by vlidholt** [comment, marked as answer]
You get this error because the database tables aren't correctly setup. In your server's `generated` folder you will find the sql code you need to run to correctly configure the tables used by Serverpod.

**Reply by JAYAMURUGANJ** [reply]
Thanks @vlidholt  its working now
